### ISSUE TITLE
This package still alive?

### ISSUE THREAD

**Issue by manuelbieh** [lwansbrough/react-native-multipeer#15]
Hey, I just discovered this package and it is exactly what I need for a project of mine. Is there any chance to get streams and resource transfer working? I would even pay some money to get this implemented 🙂

**Comment by Jonovono** [comment]
@manuelbieh what did you end up using? (if anything)

**Comment by manuelbieh** [comment]
@Jonovono Nothing yet. I was just evaluating packages in October for a feature that we're gonna implement sometime in the future, but probably not before May.

**Comment by oguzyildizz** [comment]
@manuelbieh how about now, were you able to find a reliable library?

**Comment by kutluhann** [comment]
@oguzyildiz1991 hocam, for a project we are also looking for a similar package. Were you able to find one so far?

**Comment by oguzyildizz** [comment]
@kutluhann nope, I was just looking for it for a personal project, decided to abandon it
